src/main.js

```diff
@@ -132,6 +136,9 @@ function clickHandler(event) {
   if (event.target === showRandomBtn) {
     randPoster()
   }
+  if (event.target === makePosterButton) {
```

Comment:
I know we're not "supposed to" have functions longer than 10 lines, but I would assume this is okay. Maybe we can ask an instructor on the specifics.